Sources/Bot/API/Response.swift

```diff
@@ -0,0 +1,46 @@
+
+// Based on https://github.com/mdiep/Tentacle
+
+import Foundation
+
+let linksRegex = try! NSRegularExpression(pattern: "(?<=\\A|,) *<([^>]+)>( *; *\\w+ *= *\"[^\"]+\")* *(?=\\z|,)")
+let linkParamRegex = try! NSRegularExpression(pattern: "; *(\\w+) *= *\"([^\"]+)\"")
+
+/// Returns any links, keyed by `rel`, from the RFC 5988 link header.
```

Comment:
reminds me of old days =)
https://github.com/ilyapuchka/Asana/blob/master/Asana/Data%20Access%20Layer/WebLinking.swift#L13

Reply:
Some things never change 😄

Reply:
I think I'll need to borrow that code for Stevenson at some point for multi-page requests to GitHub 😉  (That RFC5988 is a mess to parse, I still don't understand why they haven't made the standard easier to process, but 🤷‍♂)